worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/adapter/NMSRelighterFactory.java

```diff
@@ -6,7 +6,7 @@
 import com.fastasyncworldedit.core.beta.implementation.lighting.Relighter;
 import com.fastasyncworldedit.core.beta.implementation.lighting.RelighterFactory;
 import com.fastasyncworldedit.core.configuration.Settings;
-import com.fastasyncworldedit.core.object.RelightMode;
+import com.fastasyncworldedit.core.beta.implementation.lighting.RelightMode;
```

Comment:
Is it worth removing the beta package and just merging the classes/packages everywhere and inbettween?

Reply:
Preferable. The classes in named packe aren't really "beta", but what Fawe relies on on a daily basis.
I originally moved this class here because all other relight, but non nms classes, are here. Imo they would fit better in a dedicated package rather throwing everything in beta/implementation. Looking at them, we have a second `object`.

Reply:
I think:

`blocks` -> `core.queue.chunk.blocks`
`chunk` -> `core.queue.chunk`
`filter` -> `core.queue.filter`
`lighting` -> `core.extension.processor.lighting`
`packet` -> `core.util.packet`
`preloader` -> `core.util.preloader`
`processors` -> `core.extension.processor`
`queue` -> `core.queue`

The assorted classes should be quite easy to place